share/errmsg-utf8.txt

```diff
@@ -9143,6 +9143,131 @@ ER_DA_GRP_RPL_STARTED_AUTO_REJOIN
 reserved-error-section 5000 5999
 
 
+#
```

Comment:
@georgelorch @satya-bodapati Please read the commit message and confirm if you are OK with such approach - this is the best way to avoid assertions I come up with so far.